packages/web-api/src/types/response/AdminAppsActivitiesListResponse.ts

```diff
@@ -8,80 +7,81 @@
 //                                                                                     //
 /////////////////////////////////////////////////////////////////////////////////////////
 
-import type { WebAPICallResult } from '../../WebClient';
+import { WebAPICallResult } from '../../WebClient';
```

Comment:
@filmaj should know these differences. Formatting code after generating them is fine, but the process must be incorporated into the generator script.